src/block/server/componolit-interfaces-block-dispatcher.ads

```diff
@@ -120,4 +109,22 @@ is
       Pre  => Initialized (D),
       Post => Initialized (D);
 
+private
+
+   --  Enforces the precondition of Dispatch
+   --
+   --  The only valid precondition for Dispatch is Initialized (D). This enforced by this
```

Comment:
This **is** enforced

Reply:
Fixed by https://github.com/Componolit/ada-interfaces/pull/83/commits/3395b9ae71b786b4d29e9e85c56afb7e5b15568f.